### PR TITLE
New Relic Servers (add)

### PR DESCRIPTION
**Pull request by flaccid**
Most of this was pulled from the community, so I'm sure there will be changes to pretty it up, just let me know!

### CODE REVIEW

File: templates/newrelicservers/0/docker-compose.yml.tpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+newrelic-servers:
+  environment:
+    NEW_RELIC_LICENSE_KEY: ${NEW_RELIC_LICENSE_KEY}
+{{- if (.Values.FORWARD_PROXY)}}
+    FORWARD_PROXY: ${FORWARD_PROXY}
+{{- end}}
+  labels:
+    io.rancher.scheduler.global: 'true'
+    io.rancher.container.pull_image: always
```
Comment (rawmind0) [inline templates/newrelicservers/0/docker-compose.yml.tpl]: Don't need to pull image always.
```
-    io.rancher.container.pull_image: always
```

**Review by rawmind0 — CHANGES_REQUESTED** [review]

File: templates/newrelicservers/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  version: "2.3.0.132"
+  description: "Performance management solution enabling developers to diagnose and fix application performance problems in real time"
+  questions:
+  - variable: "NEW_RELIC_LICENSE_KEY"
+    label: "New Relic license key"
+    description: "Specify your New Relic license key. You can get it by clicking on 'Add more' on the servers view of newrelic."
+    type: "string"
+    default: ""
+    required: true
+  - variable: "VERSION"
```
Comment (rawmind0) [inline templates/newrelicservers/0/rancher-compose.yml]: `VERSION` variable is not used anywhere.

**Review by rawmind0 — CHANGES_REQUESTED** [review]

**Comment by rawmind0** [comment]
Hi @flaccid , thanks so much for your PR.

Please, take a look at changes requested.

**Comment by cjellick** [comment]
Very stale. Closing. Thanks! Please open a new PR if there is still a relevant change.

**Pull request closed without merging**